[6.5][ML] Update MachineLearning API documentation (#1481)
Samsung/tizen-docs: docs/application/web/guides/machine-learning/single.md

- Invoking the neural network model

>   After creating the `SingleShot` instance, you can invoke the model [synchronously](#invoke) or [asynchronously](#invoke-asynchronously) with the input data and get the inference output result.

Reviewer: Kindly ignore my previous comment, since I checked the "Master" branch for content, instead of checking in "tizen_6.5_prepare" branch. Sorry for the confusion.

    var result = model.invoke(inputTensorsData);
    ```

## Invoke asynchronously

> In case when inference takes more time it is recommended to use `invokeAsync()` method. It will help not to freeze the application UI.

Reviewer 2: ```suggestion
In case when inference takes more time it is recommended to use `invokeAsync()` method. It preserves responsive application UI during the inference.
```

> In case when inference takes more time it is recommended to use `invokeAsync()` method. It preserves responsive application UI during the inference.

Reviewer: Sure, thank you for letting know.

1. Create callbacks:

    ```javascript
    function errorCallback(error)
    {
      console.log("Error during invokeAsync: " + error.message);
    }

    function successCallback(result)
    {
      console.log("Inference finished successfully");

      /* process result here */
      console.log(result.getTensorRawData(0));

      /* always call dispose() on no longer needed objects */
      result.dispose();
    }
    ```

2. Invoke the model asynchronously:

    ```javascript
    model.invokeAsync(inputTensorsData, successCallback, errorCallback);
    ```

    > [!NOTE]

>     > Ensure to dispose the `TensorsData` object provided by successCallback, when no longer needed.

Reviewer: ```suggestion
    > Ensure to dispose the `TensorsData` object provided by `successCallback`, when no longer needed.
```

> 

Reviewer: I reviewed this file again to check for any broken links in the document. I didn't find any broken links in this file.
